packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -455,13 +455,17 @@ abstract class TransitionRoute<T> extends OverlayRoute<T> {
 /// An entry in the history of a [LocalHistoryRoute].
 class LocalHistoryEntry {
   /// Creates an entry in the history of a [LocalHistoryRoute].
-  LocalHistoryEntry({ this.onRemove });
+  LocalHistoryEntry({ this.onRemove, this.impliesAppBarDismissal = true });
```

Comment:
I think this should default to false, but i made it true for backward compatibility